src/main/groovy/wooga/gradle/appcenter/AppCenterPlugin.groovy

```diff
@@ -66,55 +65,34 @@ class AppCenterPlugin implements Plugin<Project> {
     protected static AppCenterPluginExtension create_and_configure_extension(Project project) {
         def extension = project.extensions.create(AppCenterPluginExtension, EXTENSION_NAME, DefaultAppCenterPluginExtension, project)
 
+        // TODO: Implement in a way that it handles the default value and the parsing from env/property
+//        extension.defaultDestinations.convention(AppCenterConventions.defaultDestinations.getValueProvider(project,
+//            {
+//                def result = (List<Map<String, String>> )it
+//                if (result == null){
+//                    result =it.split(',').collect { ["name": it.trim()] }
+//                }
+//                result
+//            }))
+
```

Comment:
I don't get what the issue is. The default value then must be a `String` and goes through the same mapping process as an env or property value.

Reply:
Fixed during call.